server/test/api/conventions.js

```diff
@@ -0,0 +1,73 @@
+/* global describe, it, beforeEach */
+var chai = require('chai');
+var expect = chai.expect;
+
+/** import test helpers */
+var helpers = require('./helpers');
+var uuid    = require('../../lib/guid');
+helpers.configure(chai);
+
+
+/**
+* The /cash/conventions API endpoint
+*
+* @desc This test suit is about the conventions payment in the primary cash
+*
+*/
+describe('The /cash/conventions API endpoint :: ', function () {
+  'use strict';
+
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  /** login before each request */
+  beforeEach(helpers.login(agent));
+
+  /** Test with the current date */
+  var date = new Date();
```

Comment:
It would be helpful to try posting on a previous day.  Try:
```js
var congoIndependence = new Date('1960-06-30');

// ...

var noExchangePrimaryCash = {
  uuid : uuid(),
  project_id : 1,
  /* other props ..*/
  date : congoIndependence
};
```
This should return an error -- probably that 500 error you defined in this PR.